### ISSUE TITLE
n8n Form redirect URL is not working

### ISSUE THREAD

**Issue by barn4k** [n8n-io/n8n#11657]
### Bug Description

If I set the redirect action for the new n8n form and the url is not related to the n8n site, it won't open because of the CORS policy
![image](https://github.com/user-attachments/assets/05d964c3-ee8c-45e3-b3ee-272bb8148376)

### To Reproduce

Example of the workflow:
```
{
  "meta": {
    "instanceId": "ea8ccd7cb9d9fc233a4683c7569b952c9669ba9d3ba1459d89a9944d357fa5b3"
  },
  "nodes": [
    {
      "parameters": {
        "operation": "completion",
        "respondWith": "redirect",
        "redirectUrl": "https://wikipedia.org/"
      },
      "id": "b8a1b95f-ccca-48c1-96e4-0473dc6dbf4b",
      "name": "Form",
      "type": "n8n-nodes-base.form",
      "typeVersion": 1,
      "position": [
        1580,
        680
      ],
      "webhookId": "8f2197b9-d8b9-42ec-9ae9-a16d46be1700"
    },
    {
      "parameters": {
        "formTitle": "test",
        "formFields": {
          "values": [
            {
              "fieldLabel": "label"
            }
          ]
        },
        "options": {}
      },
      "id": "f875b6c1-77f2-41ab-a89b-ef0afbea759c",
      "name": "On form submission",
      "type": "n8n-nodes-base.formTrigger",
      "typeVersion": 2.2,
      "position": [
        1300,
        680
      ],
      "webhookId": "b85acb75-c3b7-4a57-9776-ffc0d3ac3a78"
    }
  ],
  "connections": {
    "On form submission": {
      "main": [
        [
          {
            "node": "Form",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  },
  "pinData": {}
}
```

### Expected behavior

The redirect will work or It will be possible to disable CORS policy

### Operating System

cloud

### n8n Version

1.65.2

### Node.js Version

cloud

### Database

SQLite (default)

### Execution mode

main (default)

**Comment by Joffcom** [comment]
Hey @barn4k,

We have created an internal ticket to look into this which we will be tracking as "GHC-414"

**Comment by Joffcom** [comment]
Hey @barn4k,

This appears to be working for me, Can you try in production mode rather than test and let me know if that works?

**Comment by barn4k** [comment]
Same for prod url...
![image](https://github.com/user-attachments/assets/0bd9d4be-3e5c-4c1b-9ff9-4a3813def7b7)

**Comment by Joffcom** [comment]
Interesting I wonder if it is a Cloud configuration issue for the prod link, you are not loading the form in an iframe or anything are you?

**Comment by janober** [comment]
Fix got released with [`n8n@1.70.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.70.0)
